app/javascript/lib/scp-api.js

```diff
@@ -190,6 +190,70 @@ export function stringifyQuery(paramObj, sort) {
   return `?${stringified}`
 }
 
+/**
+ * Returns initial content for the upload file wizard
+ *
+ * @param {String} studyAccession Study accession
+*/
+export async function fetchStudyFileInfo(studyAccession, mock=false) {
+  const apiUrl = `/studies/${studyAccession}/file_info`
+  const [exploreInit] = await scpApi(apiUrl, defaultInit(), mock, false)
+  return exploreInit
+}
+
+/**
+ * Creates a new study file
+ *
+ * @param {String} studyId Study id
+ * @param {FormData} studyFileData html FormData object with the file data
+*/
+export async function createStudyFile(studyId, studyFileData, mock=false) {
+  const apiUrl = `/studies/${studyId}/study_files`
+  const init = Object.assign({}, defaultInit(), {
+    method: 'POST',
+    body: studyFileData
+  })
+  delete init.headers['Content-Type']
```

Comment:
Are we sure we want to delete the `Content-Type`?  Should we be setting it to `multipart/form-data` and validating that on the API side?  This seems like a possibility for something untoward to happen.

Reply:
ah, this definitely deserves a comment.  I read that this is actually the preferred thing to do so that the browser sets the type automatically with the id of the right form boundaries.  See https://stackoverflow.com/questions/36067767/how-do-i-upload-a-file-with-the-js-fetch-api (about halfway down)